hoodie-client/src/test/java/com/uber/hoodie/common/HoodieTestDataGenerator.java

```diff
@@ -209,9 +215,27 @@ public static void createSavepointFile(String basePath, String commitTime) throw
     }
   }
 
+  public static void createCleanFiles(String basePath, String commitTime) throws IOException {
+        Path commitFile =
```

Comment:
So HoodieTestUtils only creates empty files where as HoodieTestDataGenerator creates payloads. The createCleanFiles method actually writes out empty payloads and so it made sense to add it here (just like createCommit and createSavepoint).